src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$hour-height: 150px;
+$hour-width: 0.05 * $hour-height;
+$minute-height: 1.5 * $hour-height;
+$minute-width: 0.03 * $minute-height;
+$center-size: 0.1 * $hour-height;
+$clock-size: 2.5 * $minute-height;
+$border-radius: 0.3 * $hour-height;
```

Comment:
why do you think other variable are connected with `$hour-height`? set them in `px`

Reply:
fixed